consensus-peer/config.go

```diff
@@ -0,0 +1,85 @@
+/*
+Copyright IBM Corp. 2016 All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+		 http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/golang/protobuf/proto"
+	"github.com/hyperledger/fabric/consensus-peer/crypto"
+	"github.com/hyperledger/fabric/consensus-peer/persist"
+)
+
```

Comment:
I'm still a little skeptical of using JSON as our config format.  It seems like most new projects are picking yaml, and I know @corecode is not a fan, but unless we add additional pre-processing, you lose the ability to have things like comments in the config.  I know this was discussed in slack, but I never understood the reason for moving off yaml, especially if this is the format used for the rest of the fabric.